File: codehost/github/converters/comment.go
package converters

import (
	"github.com/google/go-github/v52/github"
	"github.com/reviewpal/reviewpal/codehost/target"
)

func CommentFromGithubComment(c *github.IssueComment) *target.Comment {
	return &target.Comment{
		ID:   c.GetID(),
		Body: c.GetBody(),
	}
}

File: src/codehost/github/converters/comment.rs
use octocrab::models::issues::Comment as GithubComment;
use crate::codehost::target::comment::Comment;

pub fn comment_from_github_comment(c: GithubComment) -> Comment {
    Comment {
        id: c.id.into_inner(),
        body: c.body.unwrap(),
    }
}
